styles/components/LatestNews.module.scss

```diff
@@ -0,0 +1,38 @@
+.latestNews {
+  @apply bg-color-4 grid;
+
+  .newsItem {
+    @apply grid grid-cols-5 items-center;
+    @apply m-4 p-4;
+    @apply rounded-xl;
+    &:hover {
+      @apply bg-hover-1;
+      @apply shadow-lg;
+      @apply transition-all;
```

Comment:
Can we use the specific properties here, all is slow.

Reply:
fixed